features/networking/sctp.feature

```diff
@@ -0,0 +1,59 @@
+Feature: SCTP related scenarios
+
+  # @author weliang@redhat.com
+  # @case_id OCP-28757
+  @admin
+  @destructive
+  Scenario: Establish pod to pod SCTP connections		
+    Given I store the workers in the :workers clipboard
+    And the Internal IP of node "<%= cb.workers[1].name %>" is stored in the :worker1_ip clipboard
+    
+    Given I install load-sctp-module
+    # Installing sctp module will reboot all the nodes and it will more than 10 minutes
+    And I wait up to 800 seconds for the steps to pass:
+    """
+    Given 60 seconds have passed 
```

Comment:
I think better add the following steps into `Given /^I install load-sctp-module/ do` . otherwise if you run the second `sctp` case in future.  it will always wait 60 seconds even if the the server already load sctp module. 
```
   Given 60 seconds have passed 
    When I run the :get admin command with:
      |resource | nodes |
    Then the outputs should not contain "SchedulingDisabled"
```

Reply:
It would be also better if we add a condition check in ```Given I install load-sctp-module``` like if module is already installed on the cluster, bypass this step. @weliang1 Do you think for every sctp test, loading sctp module from scratch is necessary?

Reply:
> I think better add the following steps into `Given /^I install load-sctp-module/ do` . otherwise if you run the second `sctp` case in future. it will always wait 60 seconds even if the the server already load sctp module.
> 
> ```
>    Given 60 seconds have passed 
>     When I run the :get admin command with:
>       |resource | nodes |
>     Then the outputs should not contain "SchedulingDisabled"
> ```

@zhaozhanqi Fix to have more accurate step to load and check sctp mode, 

    Given I install machineconfigs load-sctp-module   
    And I wait up to 800 seconds for the steps to pass:
    """
    Given I check load-sctp-module in <%= cb.num_workers %> workers
    """

Reply:
> It would be also better if we add a condition check in `Given I install load-sctp-module` like if module is already installed on the cluster, bypass this step. @weliang1 Do you think for every sctp test, loading sctp module from scratch is necessary?

@anuragthehatter Yes, we have that step in Given I install machineconfigs load-sctp-module. Any time the first SCTP test case need load sctp module, the rest case do not need after checking if the node has the sctp module already.

Reply:
Ah ok. Is it expected that we should check sctp module load on each worker node even if machineconfigs says sctp module exist? Also as i understand its a module and will be synced across all other nodes so checking on one node might be sufficient? hence reduce runtime.

Reply:
Would you like to have ```Given I check load-sctp-module``` code under ```Given I install machineconfigs``` step. So it could be a just one step which will install and may not install if already installed. WIll leave that upto you.

Reply:
Also line 8 and 9 should come under ```Given /^I check load-sctp-module``` step else this step always needs to accompany those two steps so better keep'em under it.WDYT

Reply:
> Ah ok. Is it expected that we should check sctp module load on each worker node even if machineconfigs says sctp module exist? Also as i understand its a module and will be synced across all other nodes so checking on one node might be sufficient? hence reduce runtime.

The whole process is like this: 1. run load sctp module, execution command finish 2. within 60 seconds, all the nodes start to reboot one by one. 3. After each node's rebooting, checking sctp module in each node is to make sure that the testing failure is not because the sctp module is not loaded in the node.
So it is expected that we should check sctp module load on each worker node.

Reply:
> > Ah ok. Is it expected that we should check sctp module load on each worker node even if machineconfigs says sctp module exist? Also as i understand its a module and will be synced across all other nodes so checking on one node might be sufficient? hence reduce runtime.
> 
> The whole process is like this: 1. run load sctp module, execution command finish 2. within 60 seconds, all the nodes start to reboot one by one. 3. After each node's rebooting, checking sctp module in each node is to make sure that the testing failure is not because the sctp module is not loaded in the node.
> So it is expected that we should check sctp module load on each worker node.

Thanks for clarifications, Weibin. Yea good to keep that since nodes are being restarted

Reply:
Actually I missed those steps in the first commit, thanks @pruan-rht for this accurate way.